conda/core/path_actions.py

```diff
@@ -566,12 +566,16 @@ class MakeMenuAction(CreateInPrefixPathAction):
     def create_actions(
         cls, transaction_context, package_info, target_prefix, requested_link_type
     ):
-        if on_win and context.shortcuts:
-            MENU_RE = re.compile(r"^menu/.*\.json$", re.IGNORECASE)
+        shorcuts_lower = [name.lower() for name in (context.shortcuts_only or ())]
+        if context.shortcuts and (
+            not context.shortcuts_only
+            or (shorcuts_lower and package_info.name.lower() in shorcuts_lower)
+        ):
+            menu_re = re.compile(r"^menu/.*\.json$", re.IGNORECASE)
```

Comment:
This is identical to the pattern below on line 1136, can we extract this into a global compiled pattern?